index.js

```diff
@@ -3,6 +3,8 @@
  */
 'use strict';
 
+// polyfills
+require('fast-text-encoding');
```

Comment:
`TextEncoder` and `TextDecoder` are already in node, so we shouldn't need this. We would, however, need some `utils.js` pattern to import them from like we do with the `crypto.js` stuff to avoid attaching them to `global` like I assume this polyfill does.

Reply:
Got a link?  I had thought something similar was in node but couldn't find it among the online pile of polyfills and similar.

Reply:
@davidlehn in terms of node I believe @dlongley  means:

Node:
https://nodejs.org/api/util.html#util_class_util_textencoder

Browser:
https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder

Node:
https://nodejs.org/api/util.html#util_new_textdecoder_encoding_options

Browser:
https://developer.mozilla.org/en-US/docs/Web/API/TextDecoder

I can add a util.js file and import them if needed

```js
export const TextEncoder = TextEncoder || require('util).TextEncoder;

export const TextDecoder = TextDecoder || require('util).TextDecoder;

```